src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/DoctrineTransportFactory.php

```diff
@@ -20,9 +21,12 @@
 
 /**
  * @author Vincent Touzet <vincent.touzet@gmail.com>
+ * @author Herberto Graca <herberto.graca@gmail.com>
  */
 class DoctrineTransportFactory implements TransportFactoryInterface
 {
+    public const DSN_PREFIX = 'doctrine://';
```

Comment:
After reading [this comment](https://github.com/symfony/symfony/pull/51034#pullrequestreview-1554567967), I'm doubting if this public constant ok, or if I should drop this commit? (only has this change)

Reply:
I would drop the constant altogether.

Reply:
Done, and tkx for the CR btw :)